.github/CODEOWNERS

```diff
@@ -71,7 +71,7 @@ clang/test/AST/Interp/ @tbaederr
 /mlir/lib/Dialect/Linalg/Transforms/ElementwiseOpFusion.cpp @MaheshRavishankar @nicolasvasilache
 /mlir/lib/Dialect/Linalg/Transforms/DataLayoutPropagation.cpp @hanhanW @nicolasvasilache
 /mlir/lib/Dialect/Linalg/Transforms/Transforms.cpp @hanhanW @nicolasvasilache
-/mlir/lib/Dialect/Linalg/Transforms/Vectorization.cpp @hanhanW @nicolasvasilache
+/mlir/lib/Dialect/Linalg/Transforms/Vectorization.cpp @banach-space @dcaballe @hanhanW @nicolasvasilache
```

Comment:
Hey @joker-eph, not sure if this is the intended behavior but it looks like if a new more specialized rule is added (e.g., `/mlir/lib/Dialect/Linalg/Transforms/Vectorization.cpp`), the existing but more generic one doesn't seem to be applied anymore (e.g., `/mlir/lib/Dialect/Linalg`). This leads to people being accidentally dropped from reviews when new rules are added if the usernames are not propagated from the generic to the specialized rules. Any thoughts about this? I couldn't find much online.